### PR TITLE
Adding basic sexual identification question.

### PR DESCRIPTION
**Pull request by nstielau**
Using adapted recommendation from "Best Practices for Asking Questions about Sexual Orientation on Surveys" [1].

[1] http://williamsinstitute.law.ucla.edu/wp-content/uploads/SMART-FINAL-Nov-2009.pdf
[2]  http://lgbtphysicists.org/files/demographics_recommendations.pdf

### CODE REVIEW

**Comment by nstielau** [comment]
Here is a more generalized version:
Do you identify with any of the following? Check all that apply.
☐ Straight
☐ Gay
☐ Lesbian
☐ Bisexual
☐ Cisgender (You are comfortable in the gender you were assigned at birth)
☐ Transgender (You are not comfortable in the gender you were assigned at birth)
☐ Self Identify: _________________

from [2].

**Comment by jakcette** [comment]
Hi. New here :) Apologies up front if this is the wrong place/way to jump in. I really like the more generalized version of this question. I know it can't cover every possible identity, but an agender option would be a nice add in my opinion.

**Comment by nstielau** [comment]
Perfect place to hop in.  I'll switch it out ;)

**Comment by nstielau** [comment]
Thanks again @jakcette, I used the recommended question from the http://lgbtphysicists.org/ recommendation.   It's funny that the UCLA recommendation clearly had a lot of small, interested people involved, but the recommendation doesn't really resonate as a useful question.

Please give any more feedback, and a '+1' if you support the addition.  

🌴

**Comment by drnikko** [comment]
I agree with @jakcette here. It would be good (and I think good for the data) to separate sexual orientation from gender identity.  We would have a section for identity that could include a "check all that apply" m, f, cis, trans, agender, gender queer, self identify, and then also one for sexual orientation, including asexual.

**Comment by nstielau** [comment]
OK, split Sexual Orientation from Gender Identification,

File: SURVEY.md [hunk 1]
```diff
@@ -90,6 +79,27 @@ Do we want people to share their disability identification? (Yes)
...
     [] Decline
 
+### Sexual Orientation
+
+  Do you identify with any of the following? Check all that apply.
+  () Straight
+  () Gay
+  () Lesbian
+  () Bisexual  
+  () Self Identify: _________________
```
Comment (nstielau) [inline SURVEY.md:89]: Queer

Comment (nstielau) [inline SURVEY.md:89]: Asexual

**Pull request merged**